programmers/creat-strange-characters.js

```diff
@@ -0,0 +1,25 @@
+function solution(s) {
+    let altrnChar; 
+    let altrnCollection;
+    s = s.split(" ");
+    for (let i = 0; i < s.length; i++) {
+        altrnCollection = [];
+        for (let j = 0; j < s[i].length; j++) {
+            if (j % 2 === 0) {
+                altrnChar = s[i][j].toUpperCase();
+            } else {
+                altrnChar = s[i][j].toLowerCase();
+            }
+        altrnCollection.push(altrnChar);
+        }
+        altrnCollection = altrnCollection.join("");
+        if (i === s.length -1) {
+            s[i] = altrnCollection;
+        } else {
+            s[i] = altrnCollection+(" ");
+        }
+    }
+    return s.join("");
+}
```

Comment:
제일 마지막에 " "(공백)이 붙어서 나오기 때문에 slice로 제일 마지막 공백을 잘라줍니다.
```suggestion
    return result.slice(0, result.length - 1);
}
```